content/TruenasApps/StableApps/ElasticSearch.md

```diff
@@ -0,0 +1,181 @@
+
+  <p style="margin-left: 33px">Create a storage dataset for Elastic Search Data Storage with a name such as <b><i>data</i></b>).
+  Select <b>apps</b> as the <b>Dataset Preset</b> for this dataset.
+  You can modify the dataset ACLs at the time of creation, or modify them later when adding them in the app.</p>
+
+<div style="margin-left: 33px">{{< include file="/static/includes/apps/BeforeYouBeginAddAppDatasetsProcedure.md" >}}</div>
+
+{{< include file="/static/includes/apps/BeforeYouBeginAddAppCertificate.md" >}}
+
+  <p style="margin-left: 33px">Adding a certificate is optional but if you want to use a certificate for this application, either create a new self-signed CA and certificate or import an existing CA and create the certificate for Elastic Search. A certificate is not required to deploy the application.</p>
+
+## Installing the Application
+
+{{< hint info >}}
+This basic procedure covers the required Elastic Search app settings.
+For optional settings, see [Understanding App Installation Wizard Settings](#understanding-app-installation-wizard-settings).
+{{< /hint >}}
+
+{{< include file="static/includes/apps/MultipleAppInstancesAndNaming.md" >}}
+
+{{< include file="/static/includes/apps/LocateAndOpenInstallWizard.md" >}}
+
+{{< trueimage src="/images/SCALE/Apps/InstallElasticSearchScreen.png" alt="Install Elastic Search Screen" id="Install Elastic Search Screen" >}}
+
+{{< include file="/static/includes/apps/InstallWizardAppNameAndVersion.md" >}}
+
+Enter the **Elastic Search Configuration** settings.
+
+Enter a password to use for the built-in `elastic` user.
+Passwords must be at least six characters long.
+
+Accept the default or enter a value in **Heap Size**.
+[Elastic documentation](https://www.elastic.co/guide/en/elasticsearch/reference/8.15/advanced-configuration.html#set-jvm-heap-size) recommends setting the heap size to no more than 50% of the total memory visible to the container.
+
+Accept the default value in **Node Name**.
+
+The TrueNAS app is configured with all the required environment variables, but if you want to further customize the container, click **Add** to the right of **Additional Environment Variables** for each to enter the variable(s) and values(s).
+
+Accept the default values in **Network Configuration**.
+See [Network Configuration](#network-configuration) for more details.
+
+Leave **Host Network** unselected.
+
+Add your **Storage Configuration** settings.
+
+Set **Host Path (Path that already exists on the system)** in **Type** for **Elastic Search Data Storage**.
+Select **Enable ACL**, and then enter or browse to and select the **data** dataset created above to populate the **Host Path** field.
+
+{{< trueimage src="/images/SCALE/Apps/InstallElasticSearchStorageDataACLandACE.png" alt="Add Elastic Search Data Storage" id="Add Elastic Search Data Storage" >}}
+
+Select **Add** to the right of **ACL Entries** for each user or group entry you want to add.
```

Comment:
As add is a button, use click not select. I'll have to go check the template to make sure it also says Click
```suggestion
Click **Add** to the right of **ACL Entries** for each user or group entry you want to add.
```